registration-processor/pre-processor/registration-processor-introducer-validator-stage/src/test/java/io/mosip/registration/processor/app/IntroducerValidatorProcessorTest.java

```diff
@@ -0,0 +1,183 @@
+import io.mosip.registration.processor.status.dto.RegistrationStatusDto;
+import io.mosip.registration.processor.status.service.RegistrationStatusService;
+
+/**
+ * The Class IntroducerValidatorProcessorTest.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ JsonUtil.class, IOUtils.class })
+@PowerMockIgnore({ "com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "javax.management.*", "javax.net.ssl.*" })
+public class IntroducerValidatorProcessorTest {
+
+	@InjectMocks
+	private IntroducerValidationProcessor introducerValidationProcessor;
+
+	@Mock
+	private Utilities utility;
+
+	/** The registration status service. */
+	@Mock
+	RegistrationStatusService<String, InternalRegistrationStatusDto, RegistrationStatusDto> registrationStatusService;
+
+	/** The audit log request builder. */
+	@Mock
+	private AuditLogRequestBuilder auditLogRequestBuilder;
+
+	/** The center validator. */
+	@Mock
+	private IntroducerValidator introducerValidator;
+
+	/** The dto. */
+	MessageDTO dto = new MessageDTO();
+
+	private String stageName;
+
+	/** The registration status dto. */
+	InternalRegistrationStatusDto registrationStatusDto = new InternalRegistrationStatusDto();
+
+	@Mock
+	private PriorityBasedPacketManagerService packetManagerService;
+
+	@Mock
+	RegistrationExceptionMapperUtil registrationStatusMapperUtil;
+
+	/**
+	 * Sets the up.
+	 *
+	 * @throws Exception the exception
+	 */
+	@Before
+	public void setUp() throws Exception {
+
+		Mockito.when(utility.getDefaultSource(any(), any())).thenReturn("reg-client");
+		Mockito.when(packetManagerService.getMetaInfo(anyString(), anyString(), any())).thenReturn(new HashMap<>());
+
+		@SuppressWarnings("unchecked")
+		RegistrationProcessorRestClientService<Object> mockObj = Mockito
+				.mock(RegistrationProcessorRestClientService.class);
+
+		Field auditLog = AuditLogRequestBuilder.class.getDeclaredField("registrationProcessorRestService");
+		auditLog.setAccessible(true);
+		auditLog.set(auditLogRequestBuilder, mockObj);
+		ResponseWrapper<AuditResponseDto> responseWrapper = new ResponseWrapper<>();
+		Mockito.doReturn(responseWrapper).when(auditLogRequestBuilder).createAuditRequestBuilder(
+				"test case description", EventId.RPR_401.toString(), EventName.ADD.toString(),
+				EventType.BUSINESS.toString(), "1234testcase", ApiName.AUDIT);
+
+		dto.setRid("123456789");
+		dto.setInternalError(false);
+		dto.setIsValid(true);
+		dto.setReg_type(RegistrationType.NEW);
+		stageName = "introducerValidatorStage";
+		registrationStatusDto = new InternalRegistrationStatusDto();
+		registrationStatusDto.setRegistrationId("123456789");
+		registrationStatusDto.setRegistrationId("reg1234");
+		Mockito.when(registrationStatusService.getRegistrationStatus(anyString())).thenReturn(registrationStatusDto);
+	}
+
+	/**
+	 * Testis valid CMD success.
+	 *
+	 * @throws Exception the exception
+	 */
+	@Test
+	public void testisValidOperatorSuccess() throws Exception {
```

Comment:
the metod calls isValidOperator, it should introducer. Very the same in all places

Reply:
changed the method name